conf/configuration.go

```diff
@@ -107,3 +118,25 @@ func LoadConfig() (Config, error) {
 
 	return cfg, nil
 }
+
+// MinPinners returns the cluster-wide value of the minimum number of servers we
+// expect to be pinning each skylink.
+func MinPinners(ctx context.Context, db *database.DB) (int, error) {
+	val, err := db.ConfigValue(ctx, confMinPinners)
+	if errors.Contains(err, mongo.ErrNoDocuments) {
+		return defaultMinPinners, nil
+	}
+	if err != nil {
+		return 0, err
+	}
+	mp, err := strconv.ParseInt(val, 10, 0)
+	if err != nil {
+		return 0, err
+	}
+	if mp < 1 || mp > 10 {
```

Comment:
f/u I'd pull this out into a const with a quick explanation on why we limit it to 10.

Reply:
By the way, the limit of 10 is purely arbitrary. If you have suggestion for another limit, I'm happy to change the value. It's just a sanity check.

Reply:
Since I needed to resolve merge conflicts anyway I just fixed this here.